src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/TracePoint.kt

```diff
@@ -184,14 +191,35 @@ internal class MethodCallTracePoint(
     val wasSuspended get() = (returnedValue == ReturnedValueResult.CoroutineSuspended)
 
     override fun toStringCompact(): String = StringBuilder().apply {
-        if (ownerName != null)
-            append("$ownerName.")
+        when {
+            isThreadCreation() -> appendThreadCreation()
+            else -> appendDefaultMethodCall()
+        }
+        appendReturnedValue()
+    }.toString()
+    
+    private fun StringBuilder.appendThreadCreation() {
+        append("thread")
+        val params = parameters?.let { getNonDefaultParametersWithName(threadParamNames, threadDefaultValues, it) } 
+            ?: emptyList()
+        if (!params.isEmpty()) {
+            append("(")
+            append(params.joinToString(", "))
+            append(")")
```

Comment:
`joinToString` already can handle such situations:

```kotlin
append(params.joinToString(", ", prefix = "(", postfix = ")"))
```

https://kotlinlang.org/api/core/kotlin-stdlib/kotlin.collections/join-to-string.html

Alternatively, you can use string template:
```kotlin
append("(${params.joinToString(", ")})")
```

Please also fix this in `appendDefaultMethodCall`.